Zend/zend_language_parser.y

```diff
@@ -827,6 +827,7 @@ return_type:
 argument_list:
 		'(' ')'	{ $$ = zend_ast_create_list(0, ZEND_AST_ARG_LIST); }
 	|	'(' non_empty_argument_list possible_comma ')' { $$ = $2; }
+	|   '(' T_ELLIPSIS ')' { $$ = zend_ast_create(ZEND_AST_CALLABLE_CONVERT); }
```

Comment:
1. What about forbidding `#[Attribute(...)]`
2. Would it make more sense to split out a value_argument_list for the first two instead and update attribute_decl, new_expr, and anonymous_class to use that instead?

(or call this argument_list_or_callable_convert)

Reply:
1. well spotted
  2. we prefer a compile error, because nicer error message ... not really sure what the words should be

Fixed :)